### PR TITLE
Showcase refresh, add Engage to header, Fast Dev video update

### PR DESCRIPTION
**Pull request by btobin**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -23,3 +23,10 @@ package-lock.json
 .vscode/
 .firebase/
 firebase-debug.log
+
+# Until future update
+Dockerfile
+docker-compose.yml
+Makefile
+.env.*
+_config_local.yml
```
Comment (johnpryan) [inline .gitignore:32]: nit: newline

**Review by johnpryan — APPROVED** [review]
LGTM

**Comment by johnpryan** [comment]
@sfshaza2 for merge

**Comment by sfshaza2** [comment]
LGTM

**Pull request merged**